compute_gnomad_phase.py

```diff
@@ -3,26 +3,79 @@
 import argparse
 from phasing import get_phased_gnomad_ht
 
+from gnomad.resources.grch37.gnomad import EXOME_POPS
+
+POPS = ["all"] + [pop.lower() for pop in EXOME_POPS]
+
+
 def main(args):
     hl.init(log="/tmp/phasing.hail.log")
 
     data_type = 'exomes' if args.exomes else 'genomes'
     path_args = [data_type, args.pbt, args.least_consequence, args.max_freq, args.chrom]
 
-    ht = hl.read_table(vp_count_ht_path(*path_args))
-    ht =  get_phased_gnomad_ht(
-        ht,
-        not args.no_em,
-        not args.no_lr,
-        not args.no_shr
-    )
+    if args.create_phased_vp_summary:
+        ht = hl.read_table(vp_count_ht_path(*path_args))
+        ht = get_phased_gnomad_ht(
+            ht,
+            not args.no_em,
+            not args.no_lr,
+            not args.no_shr
+        )
+
+        ht.write(phased_vp_count_ht_path(*path_args), overwrite=args.overwrite)
 
-    ht.write(phased_vp_count_ht_path(*path_args), overwrite=args.overwrite)
+    if args.create_phased_vp_summary_release:
+        ht = hl.read_table(phased_vp_count_ht_path(*path_args))
+        ht = ht.annotate(
+            phase_info={
+                pop: hl.struct(
+                    gt_counts=ht.phase_info[pop].gt_counts.adj,
+                    em=ht.phase_info[pop].em.adj.annotate(
+                        same_haplotype=ht.phase_info[pop].em.adj.p_chet < args.same_haplotype_em_cutoff,
+                        different_haplotype=ht.phase_info[pop].em.adj.p_chet > args.different_haplotypes_em_cutoff,
+                    )
+                )
+                for pop in POPS
+            },
+        )
+
+        ht = ht.annotate_globals(
+            max_freq=args.max_freq,
+            least_consequence=args.least_consequence,
+            same_haplotype_em_probability_cutoff=args.same_haplotype_em_cutoff,
+            different_haplotypes_em_probability_cutoff=args.different_haplotypes_em_cutoff,
+            global_annotation_descriptions=hl.struct(
+                max_freq="Maximum global variant allele frequency (adj filtered) used as the filter for inclusion in the Hail Table.",
+                least_consequence="Consequence used to determine variant inclusion in the Hail Table. The table includes all variants for which the VEP worst_consequence is at least as bad as the least_consequence. The order of consequences for this determination is is taken from gnomad_hail.constants.",
```

Comment:
```suggestion
                least_consequence="Consequence used to determine variant inclusion in the Hail Table. The table includes all variants for which the VEP worst_consequence is at least as bad as the least_consequence. The order of consequences for this determination is is taken from gnomad.utils.vep.CSQ_ORDER.",
```

This referred to the old name for gnomad_methods.